__tests__/app.test.js

```diff
@@ -76,7 +76,61 @@ describe('/api/articles/:article_id', () => {
             expect(response.body.status).toBe(400)
             expect(response.body.msg).toBe('Invalid ID parameter')
         })
-    })
+    });
+    test('PATCH 201: updates the value of votes on an article with a matching article_id', () => {
+        const patchObj = { inc_votes : 10 }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(patchObj)
+        .expect(200)
+        .then((response) => {
+            const { article } = response.body
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),
```

Comment:
We want to check specifically that the id on this is '1' - this shows we have patched the correct article